gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/ddl/catalog/GfxdSystemProcedures.java

```diff
@@ -1646,7 +1647,24 @@ public static void DROP_SNAPPY_UDF(String db, String functionName) throws SQLExc
     LeadNodeSmartConnectorOpContext ctx = new LeadNodeSmartConnectorOpContext(
         LeadNodeSmartConnectorOpContext.OpType.DROP_UDF,
         null, null, null, null, null, null, true, false, null, null,
-        db, functionName, null, null);
+        db, functionName, null, null, false, null, null, false);
+
+    sendConnectorOpToLead(ctx);
+  }
+
+  public static void ALTER_SNAPPY_TABLE(String tableIdentifier, Boolean addOrDropCol,
+                                        String columnName, String columnType,
+                                        Boolean columnNullable) throws SQLException {
+    if (GemFireXDUtils.TraceSysProcedures) {
+      SanityManager.DEBUG_PRINT(GfxdConstants.TRACE_SYS_PROCEDURES,
+              "executing ALTER_SNAPPY_TABLE ");
+    }
+
+    LeadNodeSmartConnectorOpContext ctx = new LeadNodeSmartConnectorOpContext(
```

Comment:
Apart from add/drop, add pk or attach/detach listeners when fired thorugh this interface will need changes. It is kind of different behavior right now where in those type of alter if fired thorugh the conn object will be successful as they won't be routed but here it will just fail. Please file a ticket and track this. For now we will convey to customers about this through docs.